wocky/wocky-tls-handler.c

```diff
@@ -153,15 +152,14 @@ wocky_tls_handler_init (WockyTLSHandler *self)
 static void
 real_verify_async (WockyTLSHandler *self,
     WockyTLSSession *tls_session,
-    const gchar *peername,
     GStrv extra_identities,
     GAsyncReadyCallback callback,
     gpointer user_data)
 {
   GSimpleAsyncResult *result;
   glong flags = WOCKY_TLS_VERIFY_NORMAL;
   WockyTLSCertStatus status = WOCKY_TLS_CERT_UNKNOWN_ERROR;
-  const gchar *verify_peername = NULL;
+  gchar *peername;
```

Comment:
I would prefer to keep `= NULL` initialization.
OTOH `result` above is not initialized either.

Reply:
It is actually initialized on the next row, there's no conditional return before so should be ok, peername - agree, there's conditional init (even though in the else case as well).